src/Template/Layout/default.ctp

```diff
@@ -2,7 +2,7 @@
 <html lang="<?= \Locale::getPrimaryLanguage(\Cake\I18n\I18n::locale()) ?>">
 <head>
     <?= $this->Html->charset(); ?>
-    <title><?= $this->get('title');?></title>
+    <title><?= $this->get('siteTitle');?></title>
```

Comment:
The fallback for `siteTitle` is `brand`. That's different than `title`. The `title` var is populated through `ViewListerner::_getPageTitle()` which varies depending on the current action while `siteTitle` remains same.